app.js

```diff
@@ -0,0 +1,116 @@
+var memory = {
+  initial: [],
+  operation: undefined,
+  latter: []
+}
+
+function operationAdder(div) {
+  memory.operation = this.innerHTML
+}
+
+function numberCounter(div) {
+  if( !memory.operation && (this.innerHTML !== 'AC' || '+/-' || '%' || '.' )) {
+      memory.initial.push(this.innerHTML)
+      document.querySelector('.calculator-screen').innerHTML = memory.initial.join('')
+  }else if( memory.operation && (this.innerHTML !== 'AC' || '+/-' || '%' || '.' )) {
+      memory.latter.push(this.innerHTML)
+      document.querySelector('.calculator-screen').innerHTML = memory.latter.join('')
+  }
+}
+
+document.querySelectorAll(".calculator-button").forEach(function(div) {
+  div.addEventListener('click', numberCounter )
+  div.addEventListener('click', function(){
+    if(div.innerHTML === "AC"){
```

Comment:
you should use a classname instead of the `div.innerHTML` content to determine what button was pressed